include/cartesian_geom/cartesian_kernel.h

```diff
@@ -0,0 +1,20 @@
+#include "point.h"
+
+
+#ifndef CARTESIAN_KERNEL_H
+#define CARTESIAN_KERNEL_H
+
+
+
+template <typename K>
+class Cartesian
+{
+public:
+  typedef Cartesian<K> Self;
+  typedef K                    RT;
```

Comment:
what is RT, maybe NT from Number Type?

Reply:
NT is defined as double in comp_vol.h. Maybe FT from Float Type?